openedx/core/lib/block_cache/block_structure.py

```diff
@@ -0,0 +1,591 @@
+"""
+Module with family of classes for block structures.
+    BlockStructure - responsible for block existence and relations.
+    BlockStructureBlockData - responsible for block & transformer data.
+    BlockStructureXBlockData - responsible for xBlock data.
+
+The following internal data structures are implemented:
+    _BlockRelations - Data structure for a single block's relations.
+    _BlockData - Data structure for a single block's data.
+"""
+from collections import defaultdict
+from logging import getLogger
+
+from openedx.core.lib.graph_traversals import traverse_topologically, traverse_post_order
+
+from .exceptions import TransformerException
+
+
+logger = getLogger(__name__)  # pylint: disable=C0103
```

Comment:
The pylint warning is there since `logger` is declared as a global variable and pylint wants it to be in all CAPS.  Is that what you're asking?